### DISCUSSION TITLE
Having all this CVE tracking but no confidential/private issue checkbox is bad

### DISCUSSION THREAD

**Discussion by ell1e** [community/community#5511] [Projects and Issues]
I'll just put it here, now that there is a feedback forum: having all this CVE security theater but no confidential/private issue checkbox is in my humble opinion kind of rendering GitHub's supposed security focus useless. As long as there is no "this issue should be hidden/confidential due to possible security implications" checkbox, the GitHub issue tracker is near unusable for security conscious projects. (GitLab has this checkbox, by the way.)

Some more info on this being widely wanted, but so far GitHub just seems to ignore it: https://github.com/isaacs/github/issues/37 It does feel a little like only the flashy security features that allow adding a visible `Security` tab are cared about and not actually important ones, which is disappointing. Give us useful tools, not just the ones that make the UI more marketable.

**Comment by dregad** [comment]
Duplicate of #7971 ?

**Reply by senhalil** [reply]
Well, not want to be pedantic and all but 7971 comes after 5511 so if anything 7971 could be a duplicate of 5511 but that one also talks about internal tickets and customer related information (so not really a duplicate of the security part that 5511 is focusing)

**Comment by KyeRussell** [comment]
FWIW, the new "issue forms" functionality at least allows you to prompt the user to direct security concerns elsewhere.
